### ISSUE TITLE
Fails to install on Python 3.9

### ISSUE THREAD

**Issue by jaraco** [pradyunsg/vendoring#16]
I was upgrading my pipx-managed commands today for Python 3.9 when vendoring failed to install.

```
~ $ pip -V
pip 20.2.3 from /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/pip (python 3.9)
~ $ pip install vendoring
ERROR: Could not find a version that satisfies the requirement vendoring
ERROR: No matching distribution found for vendoring
```

**Comment by pradyunsg** [comment]
AHA. It's due to the ~= 3.8.0 in the project's Python-Requires. Just a case of me being overly cautious.

**Comment by McSinyx** [comment]
*cross-linking pypa/pip#8971 so that it can be seen from there when this is fixed*

**Comment by pradyunsg** [comment]
Fixed and released as 0.3.3.
